src/main/java/io/github/thebusybiscuit/slimefun4/implementation/SlimefunPlugin.java

```diff
@@ -176,6 +177,15 @@ else if (getServer().getPluginManager().isPluginEnabled("CS-CoreLib")) {
                 return;
             }
 
+            // Disabling backwards-compatibility for fresh Slimefun installs
+            if (!new File("plugins/" + getName().replace(" ", "_"), "config.yml").exists()) {
+                config = new Config(this);
```

Comment:
This is always set to the same thing. It makes no sense for this not to be final.

Reply:
Sure, but when declaring the config, it just creates itself, so where can we check if it was created for the first time?